crates/uv/tests/lock_scenarios.rs

```diff
@@ -1733,8 +1707,7 @@ fn fork_marker_selection() -> Result<()> {
         version = "0.1.0"
         source = { editable = "." }
         dependencies = [
-            { name = "package-a", version = "0.1.0", source = { registry = "https://astral-sh.github.io/packse/0.3.29/simple-html/" } },
-            { name = "package-a", version = "0.2.0", source = { registry = "https://astral-sh.github.io/packse/0.3.29/simple-html/" } },
+            { name = "package-a" },
```

Comment:
The comment on this test says:

```rust
/// So this acts as a regression test to ensure that only one version of `a` is selected.
```

But I guess it wasn't actually doing that!